test/boost/row_cache_test.cc

```diff
@@ -4676,8 +4676,15 @@ SEASTAR_TEST_CASE(test_compact_range_tombstones_on_read) {
 // of course doesn't necessarily help release pressure on the semaphore.
 SEASTAR_THREAD_TEST_CASE(test_cache_reader_semaphore_oom_kill) {
     simple_schema s;
-    reader_concurrency_semaphore semaphore(100, 1, get_name(), std::numeric_limits<size_t>::max(), utils::updateable_value<uint32_t>(1),
-            utils::updateable_value<uint32_t>(1), reader_concurrency_semaphore::register_metrics::no);
+    reader_concurrency_semaphore semaphore(
+            100,
+            1,
+            get_name(),
+            std::numeric_limits<size_t>::max(),
+            utils::updateable_value<uint32_t>(1),
+            utils::updateable_value<uint32_t>(1),
+            utils::updateable_value<uint32_t>(1),
+            reader_concurrency_semaphore::register_metrics::no);
```

Comment:
I see a lot of cosmetic changes to the test, but I couldn't find an actual test for the new feature. Maybe it's difficult to test in a unit test? :-(

Reply:
Yes, I would need a latency test.

Reply:
We can't test that in unit tests.

I suggest to add a new class of tests (or rather - expand the scope of our weekly perf-simple-query tests). These tests would run using an already-built package on a stable machine, all alone.

We can make them part of test/perf.